plugin/pkg/scheduler/scheduler.go

```diff
@@ -269,9 +299,84 @@ func (sched *Scheduler) assume(assumed *v1.Pod, host string) error {
 	return nil
 }
 
+// assumeVolume returns true if volume binding is required
+func (sched *Scheduler) assumeVolume(assumed *v1.Pod, host string) (bool, error) {
+	// TODO: can we distingush between fully bound and waiting for binding to complete?
+	// If it's waiting for binding to complete, then we can return early here instead of
+	// spawning a go routine for bindVolumes to do nothing...
+	areVolumesBound, err := sched.config.VolumeBinder.ArePodVolumesBound(assumed)
```

Comment:
This needs to check tmpData.allbound instead in case all the pvcs got bound in the middle.